### PR TITLE
[ADD] calculation field total different

### PR DESCRIPTION
**Pull request by panharithrma**
https://trello.com/c/hUxhKuT9/232-improve-vsb-cost-adjustment

### CODE REVIEW

File: rma_vsb/models/rma_vsb_purchase_tracking.py [hunk 1]
```diff
@@ -143,11 +143,11 @@ class RmaVsbPurchaseTracking(models.Model):
         string='Promo Item - Cost', help='Promo Item - Cost', compute='compute_cost_fields')
     cost_other_costs = fields.Float(string='Other - Cost', help='Other - Cost', compute='compute_cost_fields')
     cost_3_party_com = fields.Float(
-        string='3rd Party Comm - Cost', help='Third Party Commission - Cost', compute='compute_cost_fields')
+        string='3 Party Comm - Cost', help='Third Party Commission - Cost', compute='compute_cost_fields')
```
Comment (zatinrma) [inline rma_vsb/models/rma_vsb_purchase_tracking.py:146]: Suggest : string = '3rd Party Comm - Cost'

File: rma_vsb/models/rma_vsb_purchase_tracking.py [hunk 2]
```diff
@@ -120,9 +120,9 @@ class RmaVsbPurchaseTracking(models.Model):
     est_free_service = fields.Float(string='Free Service - Est', help='Free Service - Estimated Cost', readonly=True, copy=False)
     est_promo_item = fields.Float(string='Promo Item - Est', help='Promo Item - Estimated Cost', readonly=True, copy=False)
     est_other_costs = fields.Float(string='Other - Est', help='Other - Estimated Cost', readonly=True, copy=False)
-    est_3_party_com = fields.Float(string='3rd Party Comm - Est', help='Third Party Commission - Estimated Cost', readonly=True, copy=False)
+    est_3_party_com = fields.Float(string='3 Party Comm - Est', help='Third Party Commission - Estimated Cost', readonly=True, copy=False)
```
Comment (zatinrma) [inline rma_vsb/models/rma_vsb_purchase_tracking.py:123]: Suggest : string = '3rd Party Comm - Est'

File: rma_vsb/models/rma_vsb_purchase_tracking.py [hunk 3]
```diff
@@ -102,9 +102,9 @@ class RmaVsbPurchaseTracking(models.Model):
     act_free_service = fields.Float(string='Free Service - Act', help='Free Service - Actual Cost', readonly=True, copy=False)
     act_promo_item = fields.Float(string='Promo Item - Act', help='Promo Item - Actual Cost', readonly=True, copy=False)
     act_other_costs = fields.Float(string='Other - Act', help='Other - Actual Cost', readonly=True, copy=False)
-    act_3_party_com = fields.Float(string='3rd Party Comm - Act', help='Third Party Commission - Actual Cost', readonly=True, copy=False)
+    act_3_party_com = fields.Float(string='3 Party Comm - Act', help='Third Party Commission - Actual Cost', readonly=True, copy=False)
```
Comment (zatinrma) [inline rma_vsb/models/rma_vsb_purchase_tracking.py:105]: Suggest : string = '3rd Party Comm - Act'

File: rma_account/views/rma_vsb_purchase_tracking_views.xml [hunk 4]
```diff
@@ -9,8 +9,10 @@
             <field name="arch" type="xml">
 
                 <xpath expr="//header/field[@name='vsb_status']" position="before">
-                    <button name="button_run_vsb_cost_adjustment" string="Run Adjustment" class="oe_highlight" type="object"
+                    <button name="button_run_vsb_cost_adjustment" string="Run Adjustment" class="oe_highlight"
```
Comment (zatinrma) [inline rma_account/views/rma_vsb_purchase_tracking_views.xml:12]: we should not declare button in function python function name 
Suggest name="run_vsb_cost_adjustment"

File: rma_account/models/rma_vsb_purchase_tracking.py [hunk 5]
```diff
@@ -51,37 +54,148 @@ class RmaVsbPurchaseTrackingInherit(models.Model):
...
 
-            record.adjusted_var_total = adjusted_var_total
+        return result
+    
+    @api.depends('adjustment_entry_ids.line_ids')
+    def _compute_difference_fields(self):
+        for record in self:
+            total_by_all_cost_lines = 0
+            total_diff_by_all_cost_lines = 0
+            to_adj_in_total = 0
```
Comment (zatinrma) [inline rma_account/models/rma_vsb_purchase_tracking.py:183]: can we declare to 1 line
 total_by_all_cost_lines = total_diff_by_all_cost_lines = to_adj_in_total = 0

**Review by zatinrma — APPROVED** [review]

File: rma_vsb/models/rma_vsb_purchase_tracking.py (hunk 1 above)
Comment (panharithrma) [reply rma_vsb/models/rma_vsb_purchase_tracking.py:146]: I want to diminish the number of characters to fit the form view

File: rma_account/models/rma_vsb_purchase_tracking.py (hunk 5 above)
Comment (panharithrma) [reply rma_account/models/rma_vsb_purchase_tracking.py:183]: I prefer this way, it's much cleaner and good practice

File: rma_account/views/rma_vsb_purchase_tracking_views.xml (hunk 4 above)
Comment (panharithrma) [reply rma_account/views/rma_vsb_purchase_tracking_views.xml:12]: I'll stick to the old one

**Review by zatinrma — APPROVED** [review]

**Review by zatinrma — APPROVED** [review]

**Pull request merged**